2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,22 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+
+  const average_CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [];
```

Comment:
It's better to use variable names that are just short enough here - and remember that this function doesn't know that the closing prices are from the last 5 days - they could be from the last 10 days and the function should still work! `closingPriceAverages` or `averageClosingPrices` could be a good name for the variable here.

Reply:
Thanks a lot, I will try to use those methods as many times as possible for the week4.